script/download_rust_deps.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# Copyright 2017 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+"""Script to download rust_deps."""
+
+import os
+import re
+import subprocess
+import sys
+import urllib2
+
+import deps
+
+RUST_DEPS_URL = "https://s3-us-west-2.amazonaws.com/rust-pkg-brave-core"
+RUST_DEPS_VERSION = "0.1.0"
```

Comment:
I think like this could be passed from command line.

Reply:
@bbondy actually this is called from DEPS. For now we're going to put this in a python import for both scripts and follow-up with brave/RUST_VERSION or something file

Reply:
actually I think we can control it from args.gn so let's look at that for a follow-up @tmancey

Reply:
Discussed with @bridiver and agreed we should move config to .py file as a temporary measure and then revisit to try with `custom_vars`